ang/civicase/case/actions/services/email-case-action.service.js

```diff
@@ -188,12 +188,13 @@
             action: 'add',
             hideDraftButton: 1,
             reset: 1,
-            cid: contactIDs.join(',')
+            cid: contactIDs.join(','),
+            caseid: model.caseIds[0]
           }
         };
 
-        if (model.caseIds.length === 1) {
-          popupPath.query.caseid = model.caseIds[0];
+        if (model.caseIds.length > 1) {
+          popupPath.query.allCaseIds = model.caseIds.join(',');
```

Comment:
We should not add a new parameter here(`allCaseIds`) as then there will be two parameters for the same property(case id).

Instead we should always do this is line 192
```
caseid: model.caseIds
```
then in the backend, you can check if `caseid` has a single case id or multiple ones.

And if we do this, we dont need to update any unit tests also.

Reply:
yes but the requirement is such that we need two separate params because for the first param civi creates an activity and then we have to fetch that activity through this first case id